### PR TITLE
Support for new 'preview' environment

### PR DESCRIPTION
**Pull request by leeming**
- Additional Jenkinsfile for managing deployment to dev account
  - This will be merged into the main Jenkinsfile in future iterations
  - Build to allow arbitrary branch and playbook as inputs
  - Support for mulitple 'stages' (feature directories) to be deployed at once
- Remove netlify as no longer used

### CODE REVIEW

File: slim-playbook.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    idseparator: '-@'
+    sqlpp: SQL++
+    tabs: tabs
+    toc: ~
+    page-toclevels: 1@
+    xrefstyle: short
+    enterprise: https://www.couchbase.com/products/editions[ENTERPRISE EDITION]
+    community: https://www.couchbase.com/products/editions[COMMUNITY EDITION]
+    kroki-fetch-diagram: true
+    kroki-server-url: ~
```
Comment (osfameron) [inline slim-playbook.yml:56]: We should perhaps use our internal Kroki server here

File: slim-playbook.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  - ./lib/inline-man-macro.js
+  - ./lib/multirow-table-head-tree-processor.js
+  - ./lib/swagger-ui-block-macro.js
+  - ./lib/tabs-block.js
+  - ./lib/markdown-block.js
+  - ./lib/inline-jira-macro.js
+  - asciidoctor-kroki
+ui:
+  bundle:
+    url: https://github.com/couchbase/docs-ui/releases/download/prod-135/ui-bundle.zip
```
Comment (osfameron) [inline slim-playbook.yml:69]: update to latest https://github.com/couchbase/docs-ui build (currently 165)

File: preview.Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+def awsCredentials = [$class: 'AmazonWebServicesCredentialsBinding', credentialsId: awsCredentialsId, accessKeyVariable: 'AWS_ACCESS_KEY_ID', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+//!!!
+
+def githubApiCredentials = usernamePassword(credentialsId: githubApiCredentialsId, usernameVariable: 'GITHUB_USERNAME', passwordVariable: 'GITHUB_TOKEN')
+def fontawesomeNpmTokenCredentials = string(credentialsId: fontawesomeNpmTokenCredentialsId, variable: 'FONTAWESOME_NPM_TOKEN')
+
+def triggerEventType
+def s3Cmd = 'cp --recursive'
+
+// Jenkins job configuration
```
[the remark this answers is not in the document]
Comment (osfameron) [reply preview.Jenkinsfile:23]: If not, can you update?
Would be particularly useful to see at a glance *what's the difference* between the preview and preview_multi Jenkinsfiles.

**Review by osfameron — COMMENTED** [review]
Good progress! Can you issue a PR on the docs-infra too when you get the chance?

File: preview.Jenkinsfile (hunk 3 above)
[the remark this answers is not in the document]
Comment (leeming) [reply preview.Jenkinsfile]: removed this file for now. It was more of a WIP file

File: slim-playbook.yml (hunk 1 above)
Comment (leeming) [reply slim-playbook.yml]: done

**Review by osfameron — APPROVED** [review]
Looks good to me 👍

**Pull request merged**